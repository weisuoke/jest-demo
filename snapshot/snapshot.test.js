import { generateConfig, generateAnotherConfig } from './snapshot'

test('测试 generateConfig 函数', () => {
  expect(generateConfig()).toMatchSnapshot({
    time: expect.any(Date)
  })
})

test('测试 generateAnotherConfig 函数', () => {
  expect(generateAnotherConfig()).toMatchSnapshot({
    time: expect.any(Date)
  })
})
