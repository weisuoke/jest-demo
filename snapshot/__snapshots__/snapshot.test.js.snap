// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 generateAnotherConfig 函数 1`] = `
Object {
  "domain": "localhost",
  "port": 8080,
  "server": "http://localhost",
  "time": 2021-03-14T15:02:46.998Z,
}
`;

exports[`测试 generateConfig 函数 1`] = `
Object {
  "domain": "localhost",
  "port": 8080,
  "server": "http://localhost",
  "time": Any<Date>,
}
`;
